src/sonic-yang-models/yang-models/sonic-system-ldap.yang

```diff
@@ -0,0 +1,128 @@
+module sonic-system-ldap {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-system-ldap";
+    prefix ssys;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-port {
+        prefix port;
+    }
+
+    import sonic-portchannel {
+        prefix lag;
+    }
+
+    import sonic-loopback-interface {
+        prefix loopback;
+    }
+
+    import sonic-interface {
+        prefix interface;
+    }
+
+    import sonic-mgmt_port {
+        prefix mgmt-port;
+    }
+
+    description "LDAP YANG Module for SONiC OS";
+
+    revision 2023-10-01 {
+        description "First Revision";
+    }
+
+    container sonic-system-ldap {
+
+        container LDAP_SERVER {
+            list LDAP_SERVER_LIST {
+                max-elements 8;
+                key "ipaddress";
+
+                leaf ipaddress {

```

Comment:
ip address could be a hosname as well?  ipaddress --> name ?

Reply:
modify to host, since it can support domain as well

Reply:
> done - modified to hostname, since it can support domain as well

hostname per openconfig-system.yang has the oc-inet:domain-name type which is exclusively for domain name. Since the intent for the key is to also have ipv4|ipv6 addresses then it could be suggestion to use a domain name which is not the intent. Hence the suggestion to use "name" which is neutral.

Reply:
If there is no technical problem in this yang model, I prefer to remain with the hostname, which is more descriptive of the use case.
probably someone should fix the already usage of hostname in the context of oc-inet to be domain_name to not block the usage of hostname in other cases like this one.

Reply:
hostname utility actually refers to displaying the systems DNS name (Eg: hostname cli utility). I think the oc-inet:domain-name type usage is accurate for openconfig-system.yang.